src/domains/dashboard/components/migration-manager/migration-manager.tsx

```diff
@@ -75,15 +75,15 @@ const MigrationManager = () => {
       ...userNodeAccess,
     })
 
-  const prefrenceID = migrationModalPromoInfo?.tickBoxOption.prefrenceID || ""
+  const preferenceID = migrationModalPromoInfo?.tickBoxOption.preferenceID || ""
```

Comment:
Would you mind updating `preferenceID` [here](https://github.com/netdata/dashboard/blob/ec8925a5469dbb1ec69e49bf391a2fbfff2f9e9d/src/domains/dashboard/components/migration-modal/use-migration-modal.tsx#L431) as well to avoid errors?

Reply:
oh! yeah that will be good